_posts/2017/2017-04-19-build-a-multiple-repository-pull-request-test.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: "Build a multiple repository pull request test"
+date: "2017-04-19 10:00:00"
+image: blog-header/challenge-cube.jpg
+categories: tech-stories
+authors: ["Florian Z."]
+---
+
+Did you ever face the problem of building a feature over multiple repositories?
+If you start testing the single repository, you may have unknown side effects when your code is rolled out.
+On the other hand, if you start testing the repositories together, you may have an inconsistent state of one of your code changes.
+This will lead to either errors in the test suite or wrong testing results.
+Broken tests, wrong test results and unknown side effects cause a pipeline to misses its validity.
```

Comment:
... cause a pipeline to misses its validity. > Do you mean ... cause a pipeline to miss its validity?